liquidity/liquidity.go

```diff
@@ -557,16 +605,30 @@ func (m *Manager) dispatchBestEasyAutoloopSwap(ctx context.Context) error {
 	localTotal := btcutil.Amount(0)
 	for _, channel := range channels {
 		if channelIsCustom(channel) {
-			continue
+			if assetId == "" {
```

Comment:
I think we should be able to move the new asset functionality into its own method, so we leave the legacy code be?